content/en/storage-providers/seal-workers/seal-workers.md

```diff
@@ -64,6 +64,12 @@ Ensure that workers have access to the following environment variables when they
 
 The seal workers will fail to start if the file descriptor limit is not set high enough. This limit can be raised temporarily before starting the worker by running the command `ulimit -n 1048576`. Although, we recommend setting it permanently by following the [Permanently Setting Your ULIMIT System Value]({{< relref "../../kb/soft-fd-limit/" >}}) guide.
 
+For Mac OSX, to permanently raise the limit, use:
+```bash
+sudo bash -c "echo \"limit maxfiles 1000000 1048576\" >/etc/launchd.conf"
+```
+And reload your terminal.
```

Comment:
I would suggest we move this part to also be part of the "Permanently Setting Your ULIMIT System Value" knowledge base article in /kb/soft-fd-limit.

That way we have a single article where we can list how to set it permenantly for each OS.